Woof/Woof/ViewModel/DetailPetSitterViewModel.swift

```diff
@@ -29,5 +30,13 @@ final class DetailSitterViewModel: ObservableObject {
         bio = sitter.bio
     }
 
+    /// Copies the text to the clipboard. The copied text is available in the
+    /// systemwide general clipboard, which you use for general copy-paste operations.
```

Comment:
nit:
```suggestion
    /// systemwide general clipboard used for general copy-paste operations.
```

Reply:
The apple documentation provides such a description.

Reply:
Ok, got it.